src/admin/menu/views/MenuEdit.tsx

```diff
@@ -174,31 +199,53 @@ const MenuEdit: FunctionComponent<MenuEditProps> = ({ history, match }) => {
 	) : (
 		<AdminLayout navigateBack={navigateBack} pageTitle={pageTitle}>
 			<AdminLayoutBody>
-				<Form>
-					<FormGroup label="Icoon">
-						<IconPicker
-							options={MENU_ICON_OPTIONS}
-							onChange={(e: ValueType<ReactSelectOption<string>>) =>
-								handleChange('icon', get(e, 'value', ''))
-							}
-							value={MENU_ICON_OPTIONS.find(
-								(option: ReactSelectOption<string>) => option.value === menuForm.icon
-							)}
-						/>
-					</FormGroup>
-					<FormGroup error={formErrors.label} label="Label" required>
-						<TextInput
-							onChange={(value: string) => handleChange('label', value)}
-							value={menuForm.label}
-						/>
-					</FormGroup>
-					<FormGroup error={formErrors.link} label="Link" required>
-						<TextInput
-							onChange={(value: string) => handleChange('link', value)}
-							value={menuForm.link}
-						/>
-					</FormGroup>
-				</Form>
+				<Container mode="vertical" size="small">
+					<Container mode="horizontal">
+						<Form>
+							{!menuParentId ? (
```

Comment:
`!menuParentId && (
	...
)`

Reply:
Personal preference maar ik render liever altijd null 😄

Reply:
ik gebruik && als ik slechts 1 ding wil renderen
en ? : als ik een keuze wil maken tussen 2

&& is korter :p